deepcell_label/label.py

```diff
@@ -666,30 +680,6 @@ def action_new_track(self, label):
 
         self.y_changed = self.labels_changed = True
 
-    def action_set_parent(self, label_1, label_2):
-        """
-        label_1 gave birth to label_2
-        """
-        track_1 = self.labels.tracks[label_1]
-        track_2 = self.labels.tracks[label_2]
-
-        last_frame_parent = max(track_1['frames'])
-        first_frame_daughter = min(track_2['frames'])
-
-        if last_frame_parent < first_frame_daughter:
-            track_1['daughters'].append(label_2)
-            daughters = np.unique(track_1['daughters']).tolist()
-            track_1['daughters'] = daughters
-
-            track_2['parent'] = label_1
-
-            if track_1['frame_div'] is None:
-                track_1['frame_div'] = first_frame_daughter
-            else:
-                track_1['frame_div'] = min(track_1['frame_div'], first_frame_daughter)
-
-            self.labels_changed = True
-
```

Comment:
Is this just never used?

Reply:
This is a legacy action that not used anywhere in the segmentation or tracking tools